web/src/api/typings/tasks.ts

```diff
@@ -12,11 +12,18 @@ export type Task = {
         name: string;
         id: number;
     };
-    user_id: string;
+    user: {
+        id: string;
+        name: string;
+    };
```

Comment:
we have a user object with an id field and a separate user_id field, right? what is the difference between these id?

Reply:
before changes on backend there was user_id, now user object. I created new type for task that comes from backend, so we can take user id from it and use it on front-end without changing Task type